### ISSUE TITLE
what does this app do?

### ISSUE THREAD

**Issue by leonfancy** [fmarchenko/btcloud#1]
Is this about  https://btcloud.io/ ?

**Comment by fmarchenko** [comment]
Yes, a service with similar functionality, but there is no time to finish(((

**Comment by leonfancy** [comment]
Can you give me some instructions about how to implement this? I want to build a web app like this, too.

**Comment by fmarchenko** [comment]
Sorry that took so long to answer. Of course I can tell you. What interests you?

**Comment by leonfancy** [comment]
I want to know what the technology is used to download the torrent. Do you set up a software like Transmission Daemon in the back-end?
